scipy/optimize/_highspy/meson.build

```diff
@@ -0,0 +1,82 @@
+fs = import('fs')
+if not fs.exists('../../../subprojects/highs/README.md')
+    error(
+        'Missing the `highs` submodule! Run `git submodule update --init` to fix this.',
+    )
+endif
+
+highs_opts = {
+    'default_library': 'static',
+    'with_pybind11': false,  # We don't want to build and install highspy
+    'use_zlib': 'disabled',
+    'do_install': false,
+}
+
+# Hacky workaround to not check cpu_family for 32 bit identifiers
```

Comment:
This does look hacky indeed. Don't you want `cc.sizeof('size_t') == 4` here?

Reply:
Decided to update with `sizeof(void*)` because of the minimum width of `size_t` being `16` thing [1]; also consistency, `PROPACK` uses the same check on `void*`.

[1] https://en.cppreference.com/w/c/types/size_t